examples.md

```diff
@@ -50,20 +50,32 @@ If you do not want to include them, consider to move the cache folder like below
 env:
   NUGET_PACKAGES: ${{ github.workspace }}/.nuget/packages
 steps:
-  - uses: actions/cache@v1
+  - uses: actions/cache@v2
     with:
       path: ${{ github.workspace }}/.nuget/packages
       key: ${{ runner.os }}-nuget-${{ hashFiles('**/packages.lock.json') }}
       restore-keys: |
         ${{ runner.os }}-nuget-
 ```
 
+With `actions/cache@v2` you can now exclude unwanted packages with [exclude pattern](https://github.com/actions/toolkit/tree/master/packages/glob#exclude-patterns)
```

Comment:
>"If you do not want to include them, consider to move the cache folder like below."

Should we remove the above recommendation since the wild card exclude pattern is a better approach?

Reply:
I guess it depends on how many packages the user wants to exclude. Maybe move the wild card example up to show that it's a better approach?